### DISCUSSION TITLE
Container image on Docker Hub for v2.0.3?

### DISCUSSION THREAD

**Discussion by salim-b** [formbricks/formbricks#2687] [General Requests] [answered]
Up until now I was using the official Formbricks container image [from Docker Hub](https://hub.docker.com/r/formbricks/formbricks/tags). But the latest tag there is v2.0.2 and the image hasn't been updated to the latest release v2.0.3. Is this on purpose?

I noticed that the [Formbricks docs meanwhile suggest](https://formbricks.com/docs/self-hosting/docker) to use the [container image from the GitHub registry](https://github.com/formbricks/formbricks/pkgs/container/formbricks/). Will this stay the official distribution channel for the project's container builds?

**Comment by mattinannt** [comment, marked as answer]
@salim-b There was an issue with the image build pipeline in the last release. It was fixed on the weekend and now the images should be available.
In the future we will move to Github registry for upcoming versions of Formbricks and will no longer release images to DockerHub at some point.

**Reply by salim-b** [reply]
Thanks for the explanations!
